edb/pgsql/compiler/dml.py

```diff
@@ -415,7 +415,7 @@ def process_insert_body(
                     shape_el, None, ctx=scopectx)
                 prop_elements.append(tuple_el)
 
-        valtuple = pgast.TupleVar(elements=prop_elements, named=True)
+        valtuple = pgast.TupleVarBase(elements=prop_elements, named=True)
```

Comment:
Why the change? Esp. that typically I understand a class called "*Base" to be a building block, not something to be directly instantiated.

Reply:
`TupleVarBase` differs from `TupleVar` in that the former is just a "shape" of a tuple, whereas the latter actually holds the values.  In this case we need the one without the values.  Another approach is to make `TupleElement.val` to be `Optional`, but that opens a much bigger can of worms.